src/ResourceManager/Cdn/Commands.Cdn/Microsoft.Azure.Commands.Cdn.dll-help.Help.pshproj

```diff
@@ -1940,5 +1940,179 @@
       <SupportInformation ad="false" rsat="false" ps2="false" ps3="false" ps4="false" ps5="false" wxp="false" wv="false" w7="false" w8="false" w81="false" w10="false" w2k3s="false" w2k3e="false" w2k3d="false" w2k8s="false" w2k8e="false" w2k8d="false" w2k8r2s="false" w2k8r2e="false" w2k8r2d="false" w2k12s="false" w2k12d="false" w2k12r2s="false" w2k12r2d="false" />
```

Comment:
We no longer do help this way, these changes are unnecessary.  Instead, please follow the instructions here: https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md  for creating help in Markdown